Makefile

```diff
@@ -46,10 +46,11 @@ install_tools_mac:
 	brew install yq
 	brew install helm
 
-build-cli: install_rustup_target
-ifeq ($(TARGET), armv7-unknown-linux-gnueabihf)
+ifneq (,$(findstring arm, $(TARGET))) # If TARGET contains the substring "arm"
+build-cli: install_cross
```

Comment:
I don't feel all that strongly on this but having a make variable like `CARGO_BUILDER` (not sure on the name) that's `cross` for some builds and is just `cargo` for others may be nice.

Reply:
That might be possible, yeah. Though we do need the recipe to distinguish between installing cross or a rustup target depending, so it seems like this conditional might need to stay

Reply:
Yeah. I'm noticing that the makefile conditional language is a bit lacking.